server/src/bots/telegram/botResponse/startResponse.ts

```diff
@@ -1,9 +1,33 @@
-import {getFullMenuKeyboard} from '../services/keyboard';
-import {greetUser} from '../../../messages/userMessage';
-import {showHelpInfoResponse} from './helpResponse';
-import {CallBackQueryHandler} from '../models';
+import { getFullMenuKeyboard } from '../services/keyboard';
+import { greetUser } from '../../../messages/userMessage';
+import { showHelpInfoResponse } from './helpResponse';
+import { CallBackQueryHandler } from '../models';
+import { getUser, addUser } from '../../../services/domain/user'
+import { logger } from '../../../utils/logger';
+import {User} from '../../../models/user.model'
 
 export const startResponse: CallBackQueryHandler = async (bot, message, chatId) => {
+    let user = await getUser(chatId);
+    if (!user || Object.keys(user).length === 0 ) {
+        user = {
+            chatId,
+            userName: message.chat.username || '',
+            firstName: message.chat.first_name || '',
+            lastName: message.chat.last_name || '',
+            startedOn: Date.now()
+        };
+        try {
+            await addUser(user);
+            logger.log('info', `New user ${user.chatId} was successfully added`)
```

Comment:
Use `getInfoMessage` fn in `getInfoMessage.ts` util's file

Reply:
Logging approach reset 💥